### DISCUSSION TITLE
RapidOCRPDF 默认调用的还是中文OCR 模型吗？

### DISCUSSION THREAD

**Discussion by cobaltautomationdev** [RapidAI/RapidOCRPDF#6] [Q&A] [answered]
默认的模型是中文，当识别英文文档的时候，会出现一些空格没有识别出来。可以加上paddleocr英文模型吗？或者提供一个选项让用户自己选择模型。
![image](https://github.com/RapidAI/RapidOCRPDF/assets/145321686/2052c61e-0937-4ffe-94f1-d8e7794d6df6)

**Comment by SWHL** [comment]
作为TODO，后续考虑加上

**Reply by SWHL** [reply, marked as answer]
已经在`v0.0.8`中实现，详细使用可参见[README](https://github.com/RapidAI/RapidOCRPDF#%E4%BD%BF%E7%94%A8)
